API/delivery/openapi.yaml

```diff
@@ -0,0 +1,283 @@
+              default: -1
+              maximum: 10000
+    RateableElement:
+      allOf:
+        - $ref: '#/components/schemas/Element'
+        - type: object
+          description: An element belonging to a form, that can contain multiple answer options
+          properties:
+            answerOptions:
+              type: array
+              items:
+                type: object
+                description: the answer options that rateable elements contains
+                properties:
+                  id:
+                    description: the UUID for the answer option
+                    allOf:
+                      - $ref: '#/components/schemas/Uuid'
+                  identifier:
+                    type: string
+                required:
+                  - id
+                  - identifier
+          required:
+            - answerOptions
+    ChoiceElement:
+      allOf:
+        - $ref: '#/components/schemas/RateableElement'
+        - type: object
+          description: An element belonging to a form, that can contain multiple answer options, and can allow for multiple answers to be selected by the end user
+          properties:
+            allowMultiple:
+              description: Indication whether this element allows multiple answer options to be selected
+              type: boolean
+          required:
+            - allowMultiple
+    NPSElement:
+      allOf:
+        - $ref: '#/components/schemas/RateableElement'
+        - type: object
+          description: An element that contains 11 answer options statically generated.
+          properties:
+            labelHigh:
+              $ref: '#/components/schemas/Label'
+            labelLow:
+              $ref: '#/components/schemas/Label'
+    SliderElement:
+      allOf:
+        - $ref: '#/components/schemas/Element'
+        - type: object
+          description: An element representing a set of values within a certain range
+          properties:
+            minimum:
+              type: integer
+              minimum: 0
+              maximum: 0
+            maximum:
+              type: integer
+              minimum: 1
+              maximum: 1000000
+            start:
+              type: integer
+              minimum: 0
+            labelHigh:
+              $ref: '#/components/schemas/Label'
+            labelLow:
+              $ref: '#/components/schemas/Label'
+            labelMiddle:
+              $ref: '#/components/schemas/Label'
+    FormConfiguration:
+      type: object
+      properties:
+        form:
+          type: object
+          properties: 
+            id: 
+              description: The id of the form
+              allOf:
+                - $ref: '#/components/schemas/Uuid'
+            name:
+              description: the name of the form
+              type: string
+        structure:
+          type: object
+          properties: 
+            links: 
+              type: array
+              items:
```

Comment:
We don't have a reference for the schema of this one?

Reply:
I've extracted both `link` and `version` into their own schemas.